restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/RCDeviceListener.java

```diff
@@ -91,6 +91,14 @@
     */
    void onConnectivityUpdate(RCDevice device, RCConnectivityStatus connectivityStatus);
 
+
+   /**
+    * RCDevice warning has been raised, check statusCode and statusText
+    * @param statusCode Status code
+    * @param statusText Status text
+    */
+   void onWarning(int statusCode,  String statusText);
```

Comment:
Let's also pass 'RCDevice device' as first argument per convention, so that they can easily check device status, etc

Reply:
Ok.